### PR TITLE
Angular wizard visual revamp

### PR DESCRIPTION
**Pull request by serhii-lekariev**
This PP enhances the look of the task creation wizard.

It is now centered and the buttons in the templates have been abstracted away into the parent component.

### CODE REVIEW

File: client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts [hunk 1]
```diff
@@ -107,12 +112,12 @@ export class TaskCreationWizardComponent implements AfterViewInit, OnDestroy {
...
               private readonly location: Location,
               route: ActivatedRoute,
-              private readonly navService: LayoutService) {
+              private readonly layoutService: LayoutService) {
     this.isLoading = true;
     const taskCreationId = route.snapshot.paramMap.get('taskCreationId');
     this.initWizard = wizard.init(taskCreationId);
-    this.navService.updateToolbar('Wizard');
-    this.navService.updateShowNav(false);
+    this.layoutService.updateToolbar('Wizard');
```
Comment (YegorUdovchenko) [inline client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts]: I don't think that displaying "Wizard" at the app header is a good idea. It is better to tell what is the purpose of this process.

File: client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts [hunk 2]
```diff
@@ -122,6 +127,48 @@ export class TaskCreationWizardComponent implements AfterViewInit, OnDestroy {
     throw new Error(err);
   }
 
+  /**
+   * Returns the current step of the task creation wizard.
+   *
+   * Steps are mapped 1-to-1 with stages.
+   */
+  private currentStep(): WizardStep {
+    const currentStage = this.wizard.stage;
```
Comment (YegorUdovchenko) [inline client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts]: It looks weird to have two places declaring association of the wizard stage to page. Please, try to find a better solution.

File: client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts [hunk 3]
```diff
@@ -122,6 +127,48 @@ export class TaskCreationWizardComponent implements AfterViewInit, OnDestroy {
...
+    }
+  }
+
+  /**
+   * If there are steps after the current one, moves the wizard to the next step. Finishes the
+   * wizard otherwise.
+   */
+  private proceed(): void {
+    const currentStep = this.currentStep();
+    this.isLastStage() ?
```
Comment (YegorUdovchenko) [inline client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts]: Let's try to use ternary operator for variable values assignment or for return statements, not for function invocation. Please use `if` statement here.

File: client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts [hunk 4]
```diff
@@ -189,6 +236,14 @@ export class TaskCreationWizardComponent implements AfterViewInit, OnDestroy {
   }
 
   ngOnDestroy(): void {
-    this.navService.defaultLayout();
+    this.layoutService.defaultLayout();
+  }
+
+  isLastStage(): boolean {
+    return this.wizard.stage === 3;
```
Comment (YegorUdovchenko) [inline client-angular/src/app/task-creation-wizard/task-creation-wizard.component.ts]: Looks like a "magic number". If you try to store the association of a wizard state/step in the iterable data structure then you'll be able to use more robust condition here.

**Comment by serhii-lekariev** [comment]
@YegorUdovchenko, PTAL.

**Pull request merged**